docs/running.rst

```diff
@@ -14,10 +14,23 @@ the site.
 
 The recommended ways to run Wooey are:
 
+Through two separate processes
+------------------------------
+
+You can run Wooey by calling two commands (you will need a
+separate process for each):
+
+::
+
+    python manage.py celery worker -c 1 --beat -l info
+    python manage.py runserver
+
+On Windows, the ``--beat`` option might not be supported.
```

Comment:
may doesn't imply that. May means it's a possibility that it is not supported and accepts that at a future time, it may be.